cmd/zc_traverser_local.go

```diff
@@ -73,11 +74,19 @@ func (t *localTraverser) getInfoIfSingleFile() (os.FileInfo, bool, error) {
 }
 
 func UnfurlSymlinks(symlinkPath string) (result string, err error) {
+	var count uint32
 	unfurlingPlan := []string{symlinkPath}
+	seenSymlinkPath := make(map[string]uint32)
+
+	if runtime.GOOS == "linux" {
+		return filepath.EvalSymlinks(symlinkPath)
+	}
```

Comment:
Write a comment above this to explain why we do this for Linux and not for Windows. Also the check should be != "windows" since the special handling is for windows only.

// We need to do some special UNC path handling for windows.

Reply:
Why i put only linux, as we tested only 'linux' and don't outcome on mac-os.